projects/igniteui-angular/src/lib/grids/pivot-grid/pivot-grid.component.ts

```diff
@@ -2228,12 +2229,15 @@ export class IgxPivotGridComponent extends IgxGridBaseDirective implements OnIni
                 }
             case "date":
             {
-                const dimension: IPivotDimension = {
```

Comment:
There's a bug in the `IgxPivotDateDimension` that doesn't pass the enabled property even to the root dimension it creates.